### PR TITLE
Code Coverage for ComputerInfoDebugView from #11863

### PR DESCRIPTION
**Pull request by paul1956**
Part of Fixes in https://github.com/dotnet/winforms/pull/11863

## Proposed changes
- Add Code Coverage for ComputerInfoDebugView

## Customer Impact 
- Improve Code Coverage for VB

## Regression? 

- No

## Risk
- None, tests only

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12315)

### CODE REVIEW

File: src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/ComputerInfoDebugViewTests.vb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            If PlatformDetection.IsWindows Then
+                info.AvailablePhysicalMemory.Should.NotBe(0UI)
+                info.AvailableVirtualMemory.Should.NotBe(0UI)
+                info.TotalPhysicalMemory.Should.NotBe(0UI)
+                info.TotalVirtualMemory.Should.NotBe(0UI)
+            Else
+                Assert.Throws(Of PlatformNotSupportedException)(Function() info.AvailablePhysicalMemory)
+                Assert.Throws(Of PlatformNotSupportedException)(Function() info.AvailableVirtualMemory)
+                Assert.Throws(Of PlatformNotSupportedException)(Function() info.TotalPhysicalMemory)
+                Assert.Throws(Of PlatformNotSupportedException)(Function() info.TotalVirtualMemory)
```
Comment (lonitra) [inline src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/ComputerInfoDebugViewTests.vb:26]: Can these be made into fluent assertions?

Comment (paul1956) [reply src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/ComputerInfoDebugViewTests.vb:26]: @lonitra yes I just copied that from ComputerInfo Test. Should it be fixed there as well? From what I can tell it never gets hit in either place.

Comment (paul1956) [reply src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/ComputerInfoDebugViewTests.vb]: @lonitra Fixed for ComputerInfoDebugView

**Comment by paul1956** [comment]
@lonitra I copied the platform exception from the ComputerInfoView and updated the exception testing for non Windows to be Fluent but that code is never hit.

Does this code (and the full version I copied from) ever run on a non Windows platform?

If it never does why is the "else" branch even there in both test files?

**Comment by lonitra** [comment]
Hm, I'm not aware of any test runs we have that run on a non Windows platform so I'm inclined to remove it. 🤔 Perhaps there is some historical context that @KlausLoeffelmann may know as to why that is there?

**Comment by paul1956** [comment]
> Hm, I'm not aware of any test runs we have that run on a non Windows platform so I'm inclined to remove it. 🤔 Perhaps there is some historical context that @KlausLoeffelmann may know as to why that is there?

@KlausLoeffelmann @lonitra the Non Debug View has the same code, plus not Fluent. I can remove, both it would make coverage 100% vs very low now.

**Comment by lonitra** [comment]
Let's go ahead and remove it in tests since we do not test on non Windows

**Comment by paul1956** [comment]
> Let's go ahead and remove it in tests since we do not test on non Windows

@lonitra it’s removed in my tests of ComputerInfoDebugView. It still exists in the tests for Computer Info, even if it does run on non windows platforms, the “if” is unnecessary because the excluded code should work everywhere.

**Comment by paul1956** [comment]
@lonitra what does untriaged mean?

File: src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/FileLogTraceListenerTests.vb [hunk 2]
```diff
@@ -123,9 +123,6 @@ Namespace Microsoft.VisualBasic.Forms.Tests
                         fullLogFileName = listener.FullLogFileName
                         fullLogFileName.Should.StartWith(Path.GetDirectoryName(Application.ExecutablePath))
                     End Using
-                    If File.Exists(fullLogFileName) Then
-                        File.Delete(fullLogFileName)
-                    End If
```
Comment (lonitra) [inline src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/FileLogTraceListenerTests.vb:128]: Why do we remove this?

**Review by lonitra — COMMENTED** [review]
Just a question, overall changes LGTM

**Comment by lonitra** [comment]
> what does untriaged mean?

This is only meant for issues, sometimes bot applies this after removing author feedback label that I have not been able to investigate yet..

File: src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/FileLogTraceListenerTests.vb (hunk 2 above)
Comment (lonitra) [reply src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/FileLogTraceListenerTests.vb:128]: Hm, when I run this locally I see that this get executed. Not sure why code coverage is reporting that, but I think this should be kept.

Comment (paul1956) [reply src/Microsoft.VisualBasic.Forms/tests/UnitTests/System/Windows/Forms/FileLogTraceListenerTests.vb:128]: @lonitra  I will put it back today.

**Review by lonitra — APPROVED** [review]
Thank you!

**Pull request merged**